sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -78,6 +78,7 @@ This starter provides following properties to be customized:
 | **azure.activedirectory**.authorization-clients.{client-name}.scopes | API permissions of a resource server that the application is going to acquire. |
 | **azure.activedirectory**.authorization-clients.{client-name}.scopes.on-demand | If the authorization flow of current resource server will be executed on demand. The default value is false. |
 | **azure.activedirectory**.base-uri                               | Base uri for authorization server, the default value is `https://login.microsoftonline.com/`. |
+| **azure.activedirectory**.graph-base-uri                         | Graph base uri for authorization server, the default value is `https://graph.microsoft.com/`. |
```

Comment:
If we add `graph-base-uri`, I think we should deprecate configuration item `graph-membership-uri`.

But if we deprecate `graph-membership-uri`, it's a breaking change. We should not have breaking change in minor/patch version up.

My suggestion:
1. Add new configuration item `graph-base-uri`. 
2. Add new configuration item `graph-membership-endpoint`.
3. Use (a) or (b). (a): `graph-membership-uri`. (b): Use `graph-base-uri` and `graph-membership-endpoint` to create graphMembershipUri.
3.1. Use (b): If user not configure `graph-membership-uri`, we use `graph-base-uri` and `graph-membership-endpoint` to create graphMembershipUri.  
3.2. Use (a): If user configured `graph-membership-uri`, and `graph-membership-uri` have same graph server with `graph-base-uri`. we use user-configured `graph-membership-uri`.  But giving a warn log, suggest use to use `graph-base-uri` and `graph-membership-endpoint` instead of `graph-membership-uri`.
3.3. Throw exception: If user configured `graph-membership-uri`, and `graph-membership-uri` have different graph server with `graph-base-uri`, we throw an illegalstateException.

Note: Use java code to make sure `graph-base-uri` end with `/`, but `graph-membership-endpoint` does start with `/`.

Reply:
1. If graph-base-uri is set, but graph-membership-uri is not set, graph-membership-uri is automatically generated.
2. If graph-membership-uri is set, but graph-base-uri is not set, graph-base-uri uses the default value.
3. If both graph-base-uri and graph-membership-uri are not set, both use default values.
4. If both graph-base-uri and graph-membership-uri are set, the values set by themselves are used.
5. Whether these two values are set or not, the consistency of their domains will be checked.

Reply:
`Graph base uri for authorization server` -> `Base uri for Microsoft Graph server`